sql/src/test/java/io/crate/integrationtests/SysCheckerIntegrationTest.java

```diff
@@ -74,4 +75,47 @@ public void testNumberOfPartitionCheckPassedForDocTablesCustomAndDefaultSchemas(
         SQLResponse response = execute("select severity, passed from sys.checks where id=?", new Object[]{2});
         assertThat(TestingHelpers.printedTable(response.rows()), is("2| true\n"));
     }
+
+    @Test
+    public void testUnlicencedEnterpriseDefaultIdentCheck() {
```

Comment:
Would it make sense to write unit tests for the sysCheck? This involves quite a lot of logic to test ~10 lines of code.
Maybe we could test it similar to how `CrateCircuitBreakerServiceTest´ works.